### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by shavidissa**
Update the README format and fix broken links.

### CODE REVIEW

**Comment by shavidissa** [comment]
CC @rkempf42

File: README.md [hunk 1]
```diff
@@ -1,9 +1,126 @@
...
+* [Start the DropwizardMetricsReporter](#start-the-dropwizardmetricsreporter)
+* [Stop the DropwizardMetricsReporter](#stop-the-dropwizardmetricsreporter)
+* [Types of Data You Can Report to Wavefront](#types-of-data-you-can-report-to-wavefront)
+* [Monitoring the SDK](#monitoring-the-sdk)
+* [License](#License)
+* [How to Contribute](#How-to-Contribute)
+
+# Welcome to the Wavefront Dropwizard Metrics SDK
+
+The Wavefront by Tanzu Observability by Wavefront Dropwizard Metrics SDK for Java is a library that supports reporting [Dropwizard metrics and histograms](https://metrics.dropwizard.io) to Wavefront.
```
Comment (rkempf42) [inline README.md]: The Wavefront by Tanzu Observability by Wavefront...
RK>>The Tanzu Observability by Wavefront ...

File: README.md [hunk 2]
```diff
@@ -1,9 +1,126 @@
...
+    </li>
+    </ul>
+  </td>
+</tr>
+
+</tbody>
+</table>
+
+## Prerequisites
+If you are using Maven, add the following maven dependency to your pom.xml file:
```
Comment (rkempf42) [inline README.md]: If you are using Maven, add the following maven dependency to your pom.xml file: 
>> If you are using Maven, add the following Maven dependency to your pom.xml file: 
(Backticks around file names?)

File: README.md [hunk 3]
```diff
@@ -95,15 +212,15 @@ builder.disabledMetricAttributes(ImmutableSet.<MetricAttribute>builder().
...
 
-* Otherwise, follow the steps in [Set Up a WavefrontSender](https://github.com/wavefrontHQ/wavefront-sdk-java/blob/master/docs/sender.md#set-up-a-wavefrontsender) to send data using either the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
+* Otherwise, follow the steps in [Set Up a WavefrontSender](https://github.com/wavefrontHQ/wavefront-sdk-doc-sources/blob/master/java/wavefrontsender.md#wavefrontsender) to send data using either the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
 
 
-### 4. Create a DropwizardMetricsReporter
+### Step 4: Create a DropwizardMetricsReporter
 Use the configured builder to create the `DropwizardMetricsReporter`. You must specify the `WavefrontSender` object (see above).
```
Comment (rkempf42) [inline README.md]: You must specify the `WavefrontSender` object (see above).
You must first specify the `WavefrontSender` object (see above).

File: README.md [hunk 4]
```diff
@@ -113,9 +230,9 @@ DropwizardMetricsReporter dropwizardMetricsReporter = builder.build(wavefrontSen
 ```
 ## Start the DropwizardMetricsReporter
 
-You start the `DropwizardMetricsReporter` explicitly to start reporting any metrics or histograms you create. Reporting continues until you stop the `DropwizardMetricsReporter` (see below).
+You need to explicitly start the `DropwizardMetricsReporter` to start reporting any metrics or histograms you create. Reporting continues until you stop the `DropwizardMetricsReporter` (see below).
 
-The `DropwizardMetricsReporter` reports metrics/histograms at regular intervals. You specify the length of the reporting interval to control how often data is forwarded to the `WavefrontSender`. The reporting interval determines the timestamps on the data sent to Wavefront.
+The `DropwizardMetricsReporter` reports metrics/histograms at regular intervals. You can control how often you need to send data to the `WavefrontSender` by specifying the length of the reporting interval. The reporting interval determines the timestamps on the data sent to Wavefront.
```
Comment (rkempf42) [inline README.md]: You can control how often you need to send data to the `WavefrontSender` by specifying the length of the reporting interval.
Maybe: 
 Specify the length of the reporting interval to control how often you send data to the `WavefrontSender`. ??

File: README.md [hunk 5]
```diff
@@ -15,17 +132,17 @@ Replace `$releaseVersion` with the latest version available on [maven](http://se
 
 ## Set Up a DropwizardMetricsReporter
 
-This SDK provides a `DropwizardMetricsReporter` for reporting metrics and histograms to Wavefront.
+This SDK provides a `DropwizardMetricsReporter` to report metrics and histograms to Wavefront.
 
 The steps for creating a `DropwizardMetricsReporter` are:
```
Comment (rkempf42) [inline README.md]: I prefer the phrasing: 

To create a DropwizardMetricsReporter, follow these steps:

**Review by rkempf42 — COMMENTED** [review]
Really glad this is happening! I had a few comments.

File: README.md (hunk 4 above)
Comment (shavidissa) [reply README.md]: Perfect! Thank you!

**Review by rkempf42 — APPROVED** [review]
LGTM!

**Comment by shavidissa** [comment]
CC @sushantdewan123

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**